packages/forms/src/components/ui/text-input/StandardTextInput.tsx

```diff
@@ -202,7 +206,7 @@ export const StandardTextInput: React.FC<StandardTextInputProps> = ({
     setFocused(true);
   };
 
-  const bgColorTheme = disabled
+  const resolvedBackgroundColor = disabled
```

Comment:
This are named differently throughout webui.
`backgroundColorToUse`, `activeBackgroundColor`, etc.

I think we should try to name them `active*` always.
I don't have a strong opinion on the naming, as long as it is consistent.